rasa/jupyter.py

```diff
@@ -5,10 +5,11 @@
 from rasa.core.interpreter import RasaNLUInterpreter
 from rasa.shared.nlu.interpreter import NaturalLanguageInterpreter
 from rasa.shared.utils.cli import print_error, print_success
+from rasa.core.agent import create_agent
```

Comment:
can you please reference the function by module instead of importing it directly? See [here](https://www.notion.so/rasa/Python-Code-Conventions-26835ffd72484dd699f28fa693916d2e#70d0247fdd1f48d2aec341fd91fe1dac)